scripts/Run_uncrustify_for_sources.bat

```diff
@@ -0,0 +1,91 @@
+@echo off
+setlocal
+
+rem
+rem 14.12.2016
+rem
+rem Checks if all source files conform
+rem to the uncrustify configuration
+rem The check is performed with the uncrustify
+rem program that is available in the path.
+rem This might be an older version than
+rem a newly build one.
+rem 
+rem Call this batch script from the top level
+rem directory of the uncrustify project.
+
+set SRC_DIR=src
+set OUT_DIR=results
+set CFG_FILE=forUncrustifySources.cfg
+set DIFF_FILE=lastdiff.txt
+set "PWD=%cd%"
+
+rem Check if uncrustify is available in path
```

Comment:
I am not sure about the search in the PATH var. 
The sources should be always uncrustified with a freshly build Uncrustify or else you are uncrustifying with possibly older (faulty) releases/builds. 

Relying that there is a build dir in the project dir with a freshly build Uncrustify is also not the best solution. Could this maybe be executed with CMake after a build? CMake does know where the generated binary is.